common/src/app/common/types/tokens_lib.cljc

```diff
@@ -1445,20 +1450,35 @@ Will return a value that matches this schema:
         ;; Active themes without exposing hidden penpot theme
         active-themes-clear
```

Comment:
You can name this `active-themes`. The `clear` does not add much and anyway you are removing it right after.